main.js

```diff
@@ -0,0 +1,132 @@
+import {
+  mainContainer,
+  cells,
+  tictactoebord,
+  movehtml,
+  form,
+  player1_score,
+  player2_score,
+  players,
+  startGame,
+  restart,
+} from "./dom.js"; /*eslint-disable-line */
+
+const Player = (name, mark, score) => {
+  return { name, mark, score };
+};
+
+const GameBoard = () => {
+  let moves = ["", "", "", "", "", "", "", "", ""];
+
+  const showBoard = () => {
+    return (tictactoebord.className = "board");
+  };
+
+  return { moves, showBoard };
+};
+
+const Checkwin = (board, mark) => {
+  const ismatch = (val) => val === mark;
+  return board.every(ismatch);
+};
+const WinCond = (board, mark) => {
+  const con1 = Checkwin(board.slice(0, 3), mark);
+  const con2 = Checkwin(board.slice(3, 6), mark);
+  const con3 = Checkwin(board.slice(6, 9), mark);
+  const con4 = Checkwin([board[0], board[3], board[6]], mark);
+  const con5 = Checkwin([board[1], board[4], board[7]], mark);
+  const con6 = Checkwin([board[2], board[5], board[8]], mark);
+  const con7 = Checkwin([board[0], board[4], board[8]], mark);
+  const con8 = Checkwin([board[2], board[4], board[6]], mark);
+  function checktrue(index) {
+    return index === true;
+  }
+  return [con1, con2, con3, con4, con5, con6, con7, con8].some(checktrue);
+};
+
+const gameController = (board) => {
+  const render = () => {
+    for (let index = 0; index < cells.length; index++) {
+      cells[index].innerHTML = board[index];
+    }
+  };
+
+  const makeMove = (cell, mark) => {
+    if (board[cell.id] === "") {
+      board[cell.id] = mark;
+      render();
+      return board;
+    } else {
+      return board;
+    }
+  };
+
+  const validateMove = (cells, players) => {
+    let player = 0;
+    cells.forEach((cell) => {
+      cell.addEventListener("click", function () {
+        if (cell.innerHTML.length == 0) {
```

Comment:
- any piece of code related to the view (DOM) should not be here. You need to move this code into the dom.js file. This was already pointed out in the previous review.